### PR TITLE
Various mobile improvements

### PR DESCRIPTION
**Pull request by jamesjacobs**
This branch contained various mobile improvements detailed below:

1. Modals are now larger on mobile (see screenshot1)
2. Platform specific appearance is now removed for form inputs and main search - mainly to remove the iOS inner shadow. (see screenshot 1)
3. Pagination number are hidden on mobile, previous and next are displayed. (see screenshot 2)
4. `.nav-inline` has been made 100% width.
5. `font-size` and `line-height` has been increased to match desktop. This avoids iOS zooming into the page when a form field gains focus. (see screenshot 1)
6. Sidebar help icon has been correctly aligned on mobile.  (see screenshot 3)
7. Removed blue background colour on selected piano item on mobile.  (see screenshot 4)
8. `slider` ui no longer aligns to the left on mobile.
9. Spacing on `toolbar` has been reduce on mobile. (all screenshots)
10. Tooltips have been disabled on touch screens, fixing an issue where buttons/links with tooltips would need to be tapped twice.
11. `form__indent` now aligns left on mobile.  (see screenshot 5)
12. Fixed issue where some filter bar popovers buttons would overflow their container.
13. Fixed issue where focus outline was hidden on radio and checkbox groups. Not mobile specific. (see screenshot 6)

**Modal, input and font size improvements (1, 2 and 5).**
modal-changes

**Table pagination (3)**
tabe-pagination

**Help icon alignment (6)**
help-icon

**Selected Piano item (7)**
piano-bg

**form__indent mobile alignment (11)**
form__indent

**Choice focus highlight issue (12)**
choice focus highlight

### CODE REVIEW

File: stylesheets/_component.modals.scss [hunk 1]
```diff
@@ -41,12 +41,12 @@
     margin-left: auto;
     margin-right: auto;
     max-width: 600px;
-    padding: 85px ($gutter-width / 2) 30px;
+    padding: 10px;
```
Comment (Stanton) [inline stylesheets/_component.modals.scss]: You've used `($gutter-width / 2)` below and `10px` here, explicit choice?

File: stylesheets/_component.pagination.scss [hunk 2]
```diff
@@ -15,6 +15,14 @@
   li {
     @include inline-block;
     list-style-type: none;
+
+    &:not(:first-child):not(:last-child) {
+        display: none;
```
Comment (Stanton) [inline stylesheets/_component.pagination.scss]: SR friendly?

File: stylesheets/_component.piano.scss [hunk 3]
```diff
@@ -59,10 +59,12 @@
...
 
 .piano__item.is-selected {
-    background: #e5ecf2;
-
-    &:hover {
+    @include respond-min($screen-desktop) {
         background: #e5ecf2;
+
+        &:hover {
+            background: #e5ecf2;
```
Comment (Stanton) [inline stylesheets/_component.piano.scss]: Is this in the palette? Should it be?

File: stylesheets/_component.toggle-switch.scss [hunk 4]
```diff
@@ -85,20 +85,15 @@
         color: color(white);
         content: '\F00C \F00D';
         font-family: 'FontAwesome';
-        font-size: 14px;
+        font-size: 16px;
```
Comment (Stanton) [inline stylesheets/_component.toggle-switch.scss]: `$font-size-base` ?

File: stylesheets/_component.toggle-switch.scss [hunk 5]
```diff
@@ -85,20 +85,15 @@
...
         left: 10px;
-        letter-spacing: 10px;
+        letter-spacing: 14.4px;
```
Comment (Stanton) [inline stylesheets/_component.toggle-switch.scss]: add magic number explanation in comment?

File: stylesheets/_component.pagination.scss (hunk 2 above)
Comment (jamesjacobs) [reply stylesheets/_component.pagination.scss]: Yes, I'm hiding them completely.

File: stylesheets/_component.modals.scss (hunk 1 above)
Comment (jamesjacobs) [reply stylesheets/_component.modals.scss]: Corrected.

File: stylesheets/_component.toggle-switch.scss (hunk 4 above)
Comment (jamesjacobs) [reply stylesheets/_component.toggle-switch.scss]: Corrected

File: stylesheets/_component.toggle-switch.scss (hunk 5 above)
Comment (jamesjacobs) [reply stylesheets/_component.toggle-switch.scss]: Added

**Comment by jackbentley** [comment]
How do I know what page I'm on if the pagination is only previous/next?

File: stylesheets/_component.piano.scss (hunk 3 above)
Comment (jamesjacobs) [reply stylesheets/_component.piano.scss]: swapped to var

**Comment by jamesjacobs** [comment]
@jackbentley Datatables has a breakdown above the table: `Showing 1 to 25 of 26 items`

**Pull request merged**